nixos/modules/services/networking/rxe.nix

```diff
@@ -5,19 +5,16 @@ with lib;
 let
   cfg = config.networking.rxe;
 
-  runRxeCmd = cmd: ifcs:
-    concatStrings ( map (x: "${pkgs.rdma-core}/bin/rxe_cfg -n ${cmd} ${x};") ifcs);
 
-  startScript = pkgs.writeShellScriptBin "rxe-start" ''
-    ${pkgs.rdma-core}/bin/rxe_cfg -n start
-    ${runRxeCmd "add" cfg.interfaces}
-    ${pkgs.rdma-core}/bin/rxe_cfg
-  '';
+  startScript = pkgs.writeShellScriptBin "rxe-start"
+    ( concatStringsSep "\n" ( map ( x:
+    "${pkgs.iproute}/bin/rdma link add rxe_${x} type rxe netdev ${x}" )
+    cfg.interfaces ));
 
-  stopScript = pkgs.writeShellScriptBin "rxe-stop" ''
-    ${runRxeCmd "remove" cfg.interfaces }
-    ${pkgs.rdma-core}/bin/rxe_cfg -n stop
-  '';
+  stopScript = pkgs.writeShellScriptBin "rxe-stop"
+    ( concatStringsSep "\n" ( map (x:
+    "${pkgs.iproute}/bin/rdma link delete rxe_${x}" )
```

Comment:
In theory one could pass a list to ExecStart/ExecStop. This would save the creation of two derivations and makes the service a bit more readable/easier to debug, since it contains all executed commands directly

Reply:
Nice, I did not know that works.